stable/appmesh-spire-agent/values.yaml

```diff
@@ -0,0 +1,54 @@
+# Default values for appmesh-spire-agent.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: gcr.io/spiffe-io/spire-agent
+  tag: 0.10.0
+  pullPolicy: IfNotPresent
+
+nameOverride: ""
+fullnameOverride: "spire-agent"
+
+service:
+  type: NodePort
+  port: 8081
+
+resources:
+  limits:
+    memory: 512Mi
+  requests:
+    cpu: 10m
+    memory: 1Gi
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+initContainers:
```

Comment:
add comments for the entries in values.yaml file
https://helm.sh/docs/chart_best_practices/values/#document-valuesyaml

Reply:
Sure, will do. Reason, no specific comments are added to above fields was because they are standard deployment fields.